packages/admin-ui/src/Heading/Heading.tsx

```diff
@@ -3,7 +3,7 @@ import { makeDecoratable } from "@webiny/react-composition";
 import { cva, type VariantProps } from "class-variance-authority";
 import { cn } from "~/utils";
 
-export type HeadingTags = "h1" | "h2" | "h3" | "h4" | "h5" | "h6";
+export type HeadingTags = "h1" | "h2" | "h3" | "h4" | "h5" | "h6" | "p" | "div";
```

Comment:
Why do we need to add `p` and `div` tags?

Reply:
Removed it. I think I needed it at one point, but ultimately looks like this is no longer the case. 

Thanks!